usr/lib/cca_stdll/cca_specific.c

```diff
@@ -631,113 +681,119 @@ CK_RV token_specific_rng(STDLL_TokData_t * tokdata, CK_BYTE * output,
 
 static CK_RV cca_resolve_lib_sym(void *hdl)
 {
+    #define LDSYM_VERIFY(handle, verb) \
+    *(void **)(&dll_##verb) = dlsym(handle, #verb); \
+    if ((error = dlerror()) != NULL) {             \
+        OCK_SYSLOG(LOG_ERR, "%s\n", error);        \
```

Comment:
You could also mention the verb that has failed to load in the message to syslog and trace.